lib/telnyx/verification.rb

```diff
@@ -19,9 +19,9 @@ class << self
                                     retrieve: "by_telephone",
                                   }
 
-    def self.submit_code(phone_number: nil, code: nil)
+    def self.submit_code(phone_number: nil, code: nil:, verify_profile_id: nil)
```

Comment:
you have an extra colon

`def self.submit_code(phone_number: nil, code: nil, verify_profile_id: nil)`